registration.html

```diff
@@ -0,0 +1,382 @@
+<?php
+    include "connect.php"
+?>
+
+<!DOCTYPE html>
+<html>
+
+<head>
+<<<<<<< Updated upstream
+  <title>good.teens</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
I would advise against bootstrap. Its better to learn how to quickly style things yourself. Truly mastering grid and flex will take you much further, might as well start now. 

Fully understand 
https://css-tricks.com/snippets/css/complete-guide-grid/

and 

https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
For me it is not bootstrap vs grid & flex box. It is a css framework vs custom styles. I think its better to learn styling everything yourself because if you truly want to build something custom no css framework will do and it is better to write the styles yourself. 

For this page I won't block however lets move on.